test/gen-pgbouncer-conf.sh.bats

```diff
@@ -84,12 +84,21 @@ load helper
   #
   assert grep    cognoscente    /app/vendor/pgbouncer/pgbouncer.ini
   assert grep -v arcanus        /app/vendor/pgbouncer/pgbouncer.ini
-  assert grep -v 'md5468.*559c' /app/vendor/pgbouncer/users.txt
+  assert grep -v 'md5e98.*12d3' /app/vendor/pgbouncer/users.txt
   #
   # user.txt should contain the username and the hashed form of the
   # password but not the plaintext password.
   #
   assert grep    cognoscente    /app/vendor/pgbouncer/users.txt
   assert grep -v arcanus        /app/vendor/pgbouncer/users.txt
-  assert grep    'md5468.*559c' /app/vendor/pgbouncer/users.txt
+  assert grep    'md5e98.*12d3' /app/vendor/pgbouncer/users.txt
+}
+
+@test "specific known-good username password pair" {
+  export PGBOUNCER_STATS_USERNAME=pgbouncer-stats
+  export PGBOUNCER_STATS_PASSWORD=naked
+  rm -f /app/vendor/pgbouncer/pgbouncer.ini
+  run bash bin/gen-pgbouncer-conf.sh
+  assert_success
+  assert grep 'md572c.*693' /app/vendor/pgbouncer/users.txt
```

Comment:
The credentials `pgbouncer-stats` and `naked` are what we use now.  I chose these for this test since the computed hash is available and known to be correct.

This does expose our inner credentials more widely than originally planned.  I will be changing the values used in ALI at the same time I deploy this updated heroku-buildpack-pgbouncer.